.github/workflows/sync-docs-from-node.yml

```
@@ -0,0 +1,501 @@
+            echo "api_debug_path=docs/api/debug" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Clone genlayer-node repository (docs folder only)
+        uses: actions/checkout@v4
+        with:
+          repository: genlayerlabs/genlayer-node
+          ref: ${{ steps.params.outputs.source_branch }}
+          token: ${{ secrets.NODE_REPO_TOKEN || secrets.GITHUB_TOKEN }}
+          sparse-checkout: |
+            docs
+          sparse-checkout-cone: true
```

actionlint (1.7.7)

input "sparse-checkout-cone" is not defined in action "actions/checkout@v4". available inputs are "clean", "fetch-depth", "fetch-tags", "filter", "github-server-url", "lfs", "path", "persist-credentials", "ref", "repository", "set-safe-directory", "show-progress", "sparse-checkout", "sparse-checkout-cone-mode", "ssh-key", "ssh-known-hosts", "ssh-strict", "ssh-user", "submodules", "token"

(action)